ui/ts/pages/nodes.ts

```diff
@@ -348,13 +358,15 @@ module AdminViews {
               Metrics.Select.Avg(_sysMetric("rss"))
                 .title("RSS")
             ).format(Utils.Format.Bytes).title("Memory Usage")
+              .tooltip("The memory in use across all nodes, broken down by CockroachDB Go allocations, the Go Runtime, and the total memory used by CockroachDB, including the key-value layer (RSS).")
```

Comment:
ooph. this is not very clear.
I'm not sure what the best way to put it is, but here's a summary of each of the metrics:
* rss: total memory used by the cockroach node
* go alloc: memory currently allocated by the Go layer
* go total: total memory managed by the Go allocator (includes metadata and idle memory)
* cgo alloc: memory currently allocated by the C++ layer
* go total: total memory managed by the C++ allocator (includes metadata and idle memory)